examples/gno.land/p/demo/stack/stack.gno

```diff
@@ -0,0 +1,43 @@
+package stack
+
+type Stack struct {
+	top    *node
+	length int
```

Comment:
Just to be pedantic on the types - why not make this a `unit`?
You can never have negative stack size

Reply:
Almost all of the `Len` function/method returns `int`. Just to keep it consistent with other implementations let's use 'int'.

Also, you reminded me of this issue https://github.com/golang/go/issues/27460